api/envoy/data/accesslog/v3/accesslog.proto

```diff
@@ -44,6 +44,9 @@ enum AccessLogType {
   UpstreamPeriodic = 8;
   UpstreamEnd = 9;
   DownstreamTunnelSuccessfullyEstablished = 10;
+  UdpTunnelUpstreamConnected = 11;
```

Comment:
can you please also update the access log docs with description of these values?